source/components/bc_component.cpp

```diff
@@ -132,6 +132,16 @@ unitval BlackCarbonComponent::getData( const std::string& varName,
     return returnval;
 }
 
+void BlackCarbonComponent::reset(double time) throw(h_exception)
+{
+    // Set time counter to requested date; there are no outputs to
+    // reset.
+    oldDate = time;
+    H_LOG(logger, Logger::NOTICE)
+        << getComponentName() << " resetting to time= " << time << "\n";
```

Comment:
All the other messages say "reset to time" - let's be consistent

Reply:
Ok, done.